front/app/utils/dateUtils.ts

```diff
@@ -77,6 +77,21 @@ export function timeAgo(dateInput: number, locale: SupportedLocale) {
 
     if (value <= Math.abs(secondsElapsed)) {
       const delta = secondsElapsed / value;
+
+      // Special handling for months: if it's near 12 months or multiples of 12 months,
```

Comment:
I think it's easier to understand if you first understand the improved version of this function: https://github.com/CitizenLabDotCo/citizenlab/pull/10476/files